### PR TITLE
#92 Update docs with Java/JavaFX 12

### PR DESCRIPTION
**Pull request by jperedadnr**
#89 Add JavaFX maven archetypes
#86 Add JavaFX maven plugin and #76 remove exec plugin

### CODE REVIEW

**Comment by abhinayagarwal** [comment]
At some point of time, we need to use a placeholder for Java/FX version so that we don't have to change all the files every 6 months :)

File: IDE/IntelliJ/Non-Modular/Maven/hellofx/pom.xml [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
     <version>1.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
-
-    <name>hellofx</name>
-
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <mainClass>org.openjfx.MainApp</mainClass>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```
[the remark this answers is not in the document]
Comment (jperedadnr) [reply IDE/IntelliJ/Non-Modular/Maven/hellofx/pom.xml]: There is no particular reason, but for the samples it might be better to align everything to 12.

**Comment by jperedadnr** [comment]
That would be ideal, but in this case we are changing also the plugins, the archetypes (including code base), and there are IDE important changes as well...

File: IDE/IntelliJ/Non-Modular/Maven/hellofx/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jperedadnr) [reply IDE/IntelliJ/Non-Modular/Maven/hellofx/pom.xml]: Updated

**Pull request merged**